projects/create-and-vary-a-licence-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/model/ManagedOffender.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.api.model
+
+import java.time.LocalDate
+
+data class ManagedOffender(
+    val offenderCrn: String,
+    val offenderName: Name,
```

Comment:
We probably don't need the `offender` prefix here
```suggestion
    val crn: String,
    val name: Name,
```

Reply:
Done